GTG/gtk/browser/tag_editor.py

```diff
@@ -199,7 +199,7 @@ def __init__(self, req, app, tag=None):
         # Build up the window
         self.set_type_hint(Gdk.WindowTypeHint.DIALOG)
         self.set_position(Gtk.WindowPosition.CENTER)
-        self.set_title('Edit tag')
+        self.set_title(_('Editing Tag "%s"') % tag.get_name())
```

Comment:
Unicode quotation should be used, but that may be a separate thing to attack throughout the app separately (and in fact doing this single one alone here might make it annoyingly inconsistent).
https://developer.gnome.org/hig/stable/typography.html.en